golden/bullseye.pkr.hcl

```diff
@@ -0,0 +1,50 @@
+variables {
+  headless          = true
+  iso_checksum      = "224cd98011b9184e49f858a46096c6ff4894adff8945ce89b194541afdfd93b73b4666b0705234bd4dff42c0a914fdb6037dd0982efb5813e8a553d8e92e6f51"
+  iso_checksum_type = "sha512"
+  iso_url           = "https://cdimage.debian.org/debian-cd/11.6.0/amd64/iso-cd/debian-11.6.0-amd64-netinst.iso"
+}
+
+source "virtualbox-iso" "base-debian-amd64" {
+  boot_command         = [
+        "<esc><wait>",
+        "install <wait>",
+        " auto=true",
+        " priority=critical",
+        " preseed/url=http://{{ .HTTPIP }}:{{ .HTTPPort }}/preseed.cfg PACKER_USER=packer PACKER_AUTHORIZED_KEY={{ .SSHPublicKey | urlquery }}<enter>",
+	"<enter><wait>"
+  ]
+
+  http_directory       = "/home/sdake/images/cfg"
+  cpus                 = 8
+  disk_size            = 65535
+  guest_os_type        = "Debian11_64"
+  guest_additions_mode = "attach"
```

Comment:
We don't need guest additions. You can remove it now or later. If later keep track of the tech debt in a new github issue so others in our team can fix these low hanging fruit as they come up to speed.